src/util/std_code.h

```diff
@@ -2380,6 +2380,59 @@ inline code_try_catcht &to_code_try_catch(codet &code)
   return ret;
 }
 
+/// \ref codet representation of a 'havoc_object' statement.
```

Comment:
Can we please refrain from recursive documentation? No B. means B., please. If there genuinely is documentation of said statement then refer to it, else here will need to be the place to document it, I'm afraid.